crates/anvil/README.md

```diff
@@ -11,6 +11,12 @@ A local Ethereum node, akin to Ganache, designed for development with [**Forge**
     -   mining modes: auto, interval, manual, none
     -   ...
 
+## Supported Versions
+
+- **solc**: 0.8.25
+- **evm**: Cancun
+
+  
```

Comment:
this should mention anvil + forge
forge is on latest solc + cancun
anvil is on cancun

Reply:
> this should mention anvil + forge forge is on latest solc + cancun anvil is on cancun

ty very much for your review, does this looks better for you ?